pkgs/by-name/au/authentik/package.nix

```diff
@@ -186,76 +182,87 @@ let
-          argon2-cffi
-          celery
-          channels
-          channels-redis
-          codespell
-          colorama
-          dacite
-          deepmerge
-          defusedxml
-          django
-          django-cte
-          django-filter
-          django-guardian
-          django-model-utils
-          django-pglock
-          django-prometheus
-          django-redis
-          django-storages
-          django-tenants
-          djangorestframework
-          djangorestframework-guardian2
-          docker
-          drf-spectacular
-          duo-client
-          facebook-sdk
-          fido2
-          flower
-          geoip2
-          google-api-python-client
-          gunicorn
-          jsonpatch
-          kubernetes
-          ldap3
-          lxml
-          msgraph-sdk
-          opencontainers
-          packaging
-          paramiko
-          psycopg
-          pydantic
-          pydantic-scim
-          pyjwt
-          pyyaml
-          requests-oauthlib
-          scim2-filter-parser
-          sentry-sdk
-          service-identity
-          setproctitle
-          structlog
-          swagger-spec-validator
-          tenant-schemas-celery
-          twilio
-          twisted
-          ua-parser
-          urllib3
-          uvicorn
-          watchdog
-          webauthn
-          wsproto
-          xmlsec
-          zxcvbn
-        ]
-        ++ channels.optional-dependencies.daphne
-        ++ django-storages.optional-dependencies.s3
-        ++ opencontainers.optional-dependencies.reggie
-        ++ psycopg.optional-dependencies.c
-        ++ uvicorn.optional-dependencies.standard;
+        nativeBuildInputs = [
+          prev.poetry-core
+          final.pythonRelaxDepsHook
+        ];
+
+        propagatedBuildInputs =
+          with final;
+          [
+            argon2-cffi
+            celery
+            channels
+            channels-redis
+            cryptography
+            dacite
+            deepmerge
+            defusedxml
+            django
+            django-countries
+            django-cte
+            django-filter
+            django-guardian
+            django-model-utils
+            django-pglock
+            django-prometheus
+            django-redis
+            django-storages
+            django-tenants
+            djangorestframework
+            djangorestframework-guardian2
+            docker
+            drf-orjson-renderer
+            drf-spectacular
+            duo-client
+            fido2
+            flower
+            geoip2
+            google-api-python-client
+            gunicorn
+            gssapi
+            jsonpatch
+            jwcrypto
+            kubernetes
+            ldap3
+            lxml
+            msgraph-sdk
+            opencontainers
+            packaging
+            paramiko
+            psycopg
+            pydantic
+            pydantic-scim
+            pyjwt
+            pyrad
+            python-kadmin-rs
```

Comment:
In 2024.10, we depend on [python-kadmin](https://github.com/authentik-community/python-kadmin.git), which we're replacing with python-kadmin-rs in 2024.12

Reply:
right, I was wondering if I did a mistake. I would say let's wait for 2024.12 and I will update this PR then

Reply:
Oh I didn't see the `pythonRemoveDeps`, my bad. Nonetheless, it's not a drop-in replacement, so it might not work.

Also `pkgs.krb5` should be added as a runtime dep.

Reply:
`pkgs.krb5` is a depdency of `python-kadmin-rs` so it should already be included, right?

Reply:
True, I wasn't sure if those were propagated to runtime or not.